metadb/scrapers/musicbrainz.py

```diff
@@ -0,0 +1,44 @@
+import getpass as gt
+import sys
+import musicbrainzngs as mb
+
+#username = raw_input('Enter MusicBrainz Username: ')
+#password = gt.getpass('Password for {}: '.format(username))
+#mb.auth(username,password)
+
+mb.set_useragent(
+    "python-metadb-musicbrainz-scrape",
+    "0.1",
+    "https://github.com/MTG/metadb")
+
+def scrape(mbid):
+    try:
+        result = mb.get_recording_by_id(mbid, includes=["artists", "releases", "tags"])
+        
+        recording_name = result["recording"]["title"] #recording title
+        
+        artists = []
+        
+        for item in result["recording"]["artist-credit"]:
+            if isinstance(item, dict):
+                artists.append({"id": item["artist"]["id"], "name": item["artist"]["name"]})
+        
+        releases = []
+        release_groups = []
+        
+        for item in result["recording"]["release-list"]:
+            releases.append({"id": item["id"], "Title": item["title"], "Date": item["date"]})
```

Comment:
we should use completely lower-case dictionary keys here - note that all of the responses from the musicbrainz library are like this too